lib/node/runner/checker_ballot_transaction.go

```diff
@@ -227,6 +233,10 @@ func ValidateOp(st *storage.LevelDBBackend, source *block.BlockAccount, op trans
 		if casted.Linked != "" && (casted.Amount%common.Unit) != 0 {
 			return errors.ErrorFrozenAccountCreationWholeUnit // FIXME
 		}
+		if casted.Linked != "" && casted.Linked != tx.Source() {
+			err = errors.ErrorFrozenAccountMustCreatedFromLinkedAccount
```

Comment:
Why ?

Reply:
@Geod24 I think this is needed for prevent mistake of user making link to other account.

Reply:
The node itself should not enforce this. because it has nothing to do with the ability of the node to make consensus. There is nothing definitely wrong with a frozen account benefitting someone else. If we want to enforce such a thing, it should be done client side.

Moreover, I disagree that the mistake is easy to make. The user has to put a valid account address in there. So simple errors like typos will get caught.

Reply:
@Geod24 Ok I agree that it should be done client side. I remove it.